### PR TITLE
Features/enhanced readme

### PR DESCRIPTION
**Pull request by mayaalsamarrai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,78 +1,104 @@
...
+![Fig.Recurring order](docs/media/screen-recurring-order.png)
+1. Customer sets the recurrence parameters:
+![Fig. Autoorder](docs/media/screen-autoreorder.png)
+1. Customer proceeds with the checkout and submits the order;
+1. System creates both the order and a subscription based on that order;
+1. System background job checks periodically the existing subscriptions and generates new orders (if needed);
+1. System sends email notification to the customer about the new order;
+1. Customer receives the email and clicks the link to open the order in storefront;
+1. a) Customer reviews the order and confirms payment;
+1. b) Alternatively, customer opens the associated subscription and cancels it. No more new orders would be generated for this subscription ever again.
```
Comment (mvktsk) [inline README.md]: List marking error 
9. a)
10. b)

File: README.md [hunk 2]
```diff
@@ -1,78 +1,104 @@
...
-3. Customer sets the recurrence parameters
-4. Customer proceeds with the checkout and submits the order
-5. System creates both the order and a subscription based on that order
-6. System background job periodically checks existing subscriptions and generates new orders (if needed)
-7. System sends email notification to the customer about the new order
-8. Customer receives the email and clicks the link to open the order in storefront
-9. a) Customer reviews the order and confirms payment
-9. b) Alternatively, customer opens the associated subscription and cancels it. No more new orders would be generated for this subscription ever again.
+* Notifications for each life phase and subscription events
+* Possibility to edit notification templates and manage the list of notification types
```
Comment (mvktsk) [inline README.md:22]: Do we need to add in Key features somthing like "Using templates for notifications" ?

File: README.md [hunk 3]
```diff
@@ -1,78 +1,104 @@
...
 
 ![Recurring order workflow](https://cloud.githubusercontent.com/assets/5801549/21717221/4dace7d0-d418-11e6-8688-56866b71be27.png)
 
-## Subscription scenario
-1. Customer Service Representative (CSR) creates a new product and defines auto-order schedule for it.
-2. Customer orders such product as usual. Note: there will be no possibility to select "I want this to be a recurring order" during checkout.
-3. The remaining steps are identical to the ones in "Recurring order scenario" starting from step 5.
+### Subscription scenario
+
+1. Customer Service Representative (CSR) creates a new product and defines auto-order schedule for it on admin side;
```
Comment (mvktsk) [inline README.md]: Link to CSR description

File: README.md [hunk 4]
```diff
@@ -1,78 +1,104 @@
...
+![Fig. Autoreorder schedule](docs/media/screen-autoorder-schedule.png)
+1. Customer orders the product on storefront
```
Comment (mvktsk) [inline README.md]: **;** missed at the end of string

File: README.md [hunk 5]
```diff
@@ -1,78 +1,104 @@
...
+1. The system displays the subscription schedule under the product;
+![Fig. Subscription period](docs/media/screen-subscription-period.png)
+1. There will be no possibility to select "I want this to be a recurring order" during checkout.
```
Comment (mvktsk) [inline README.md]: change **.** to **;** at the end of string

File: README.md [hunk 6]
```diff
@@ -1,78 +1,104 @@
...
-* When a Customer creates an Order in storefront, it's sent to Orders service;
-* Orders service saves the Customer Order to database and also fires OrderChangeEvent even;
-* Subscription module receives OrderChangeEvent (containing the Order).
-    * If this was a newly created Order with associated payment plan (and not a prototype for some Subscription), then:
-    	1. Create new Subscription;
-    	2. Clone given Order and associate it with the new Subscription.
-    * If the received Order was generated by a Subscription, then the Subscription should be updated (status, balance recalculated, etc.). This includes the scenario when new Order was generated by Process Subscription Job.
-* Scheduled job initiates Subscription processing periodically. It reads active Subscriptions from database and generates new Orders (if needed). 
+1. When a Customer creates an Order on storefront, it's sent to Orders service;
+1. Orders service saves the Customer Order to database and also fires; OrderChangeEvent even;
```
Comment (mvktsk) [inline README.md]: >  fires; OrderChangeEvent **even**;
Typo, should be **fires** OrderChangeEvent even**t**; ?

File: README.md [hunk 7]
```diff
@@ -1,78 +1,104 @@
...
+1. Subscription module receives OrderChangeEvent (containing the Order);
+1. If this was a newly created Order with associated payment plan (and not a prototype for some Subscription), then:
+
+     1. Create a new Subscription-
+    clone given Order and associate it with the new Subscription;
```
Comment (mvktsk) [inline README.md]: merge 70 and 71 into one string
space missing after **Subscription**

File: README.md [hunk 8]
```diff
@@ -1,78 +1,104 @@
...
+     1. If the received Order was generated by a Subscription, then the Subscription should be updated (status, balance recalculated, etc.). This includes the scenario when new Order was generated by Process Subscription Job.
+
+1. Scheduled job initiates Subscription processing periodically. It reads active Subscriptions from database and generates new Orders (if needed).
```
Comment (mvktsk) [inline README.md]: Describe more about scheduled job

**Review by mvktsk — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mayaalsamarrai) [reply README.md:22]: May be we do!

File: README.md (hunk 3 above)
Comment (mayaalsamarrai) [reply README.md]: What do you mean?

File: README.md (hunk 6 above)
Comment (mayaalsamarrai) [reply README.md]: May be:)

File: README.md (hunk 8 above)
Comment (mayaalsamarrai) [reply README.md]: Please give me info, what exactly we should describe ?

File: README.md (hunk 3 above)
Comment (mvktsk) [reply README.md]: Link to article (if exists) where describes What CSR is?

File: README.md (hunk 8 above)
Comment (mvktsk) [reply README.md]: This is the first place in the article where “scheduled job” is mentioned. It is not clear who and when “scheduled job” created.

File: README.md (hunk 3 above)
Comment (mayaalsamarrai) [reply README.md]: Customer Service Rep is just a role, like Content manager or whatever, so there is no article or any special description we can refer to

**Review by mvktsk — APPROVED** [review]

**Pull request merged**